<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>750</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cañon 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>posicion en x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>posicion en y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cañon 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="posx1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>600</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="posx2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>600</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="posy1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>600</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="posy2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>600</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Disparo_O">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>510</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disparo Ofensivo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
